res/themes/light/css/_light.scss

```diff
@@ -14,7 +14,7 @@ $monospace-font-family: Inconsolata, Twemoji, 'Apple Color Emoji', 'Segoe UI Emo
 
 // unified palette
 // try to use these colors when possible
-$accent-color: #0DBD8B;
+$accent-color:  #002677;
```

Comment:
Perhaps externalise this into our copy of the light theme?  
Otherwise, as mentioned in another conversation, please implement all changes as either:
1. Controlled by a config.json configuration property that defaults to the current value in the element code, if the propoerty is not in config.json
2. We have our own copy of the file in aether-host-files/common/element-web/matrix-react-sdk (this is the least favoured solution as it doesn't address the merge issues with element master code, but has been done for a quick tactical solution)

Reply:
done